src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLabel.vue

```diff
@@ -3,8 +3,8 @@
     title="Double click to edit"
     class="px-1 cursor-text right hover:text-skin-message-hover hover:bg-skin-message-hover"
     :class="{
-      'py-1 px-2 ml-1 cursor-text': editing,
-      'absolute right-1/2 translate-x-1/2 bottom-0': editing && !isSelfAsync,
```

Comment:
I don't know why we needed this either.

Reply:
This makes sure the text will always stay center when editing.

Reply:
Without those classes, the message text always stays at its original position and when we commit the changes the message is also staying where it is. I feel that is a better experience.

Reply:
I want to mention that using `position: absolute;` is essential for editing text. Without it, the arrow may get displaced by long text. With position set to absolute, centering the editing text becomes much simpler compared to maintaining its original position.